.travis.yml

```diff
@@ -1,24 +0,0 @@
-sudo: no
-language: node_js
-node_js:
-- 'lts/*'
-- 'node'
-branches:
-  only:
-  - master
-install:
-- npm install
-- npm install jshint
-script:
-- jshint .
-- npm test
-notifications:
-  email: false
-deploy:
-  provider: npm
-  email: lonnen@mozilla.com
-  api_key:
-    secure: a+RxKYrMVIA7B/SiphMVLBVlaeuufTwas6bdUlYCJWlzMugtKBDqnYu4NN9qTdQIOmy5Z0zgnOxvf8VQEfLQ3B3rJTw6/JSl3FlSfmoiWmTxRJx4MCYXpJykAgvdZeKIysG91watX/BQRYS3LR8KkxER/coqiLoOMtZwAfrQO2Y=
```

Comment:
Lonnen has confirmed that he's deleted this token from NPM

Reply:
yep. Token rotated and activity audited